init/module-setup.sh

```diff
@@ -25,6 +25,11 @@ kdump_check_net() {
     elif [ "${KDUMP_NETCONFIG%:force}" != "$KDUMP_NETCONFIG" ]; then
         # always set up network
         kdump_neednet=y
+    elif [ -f "/usr/lib/fence_kdump_send" ] &&
+         ( [[ $KDUMP_PRESCRIPT =~ "fence_kdump_send" ]] || \
+         [[ $KDUMP_POSTSCRIPT =~ "fence_kdump_send" ]] ) ; then
+        # setup network when fence_kdump_send included and configured
+        kdump_neednet=y
```

Comment:
We should also add $FENCE_KDUMP_SEND to KDUMP_REQUIRED_PROGRAMS here. This does not work for a number of reasons, e.g. it would not be added if KDUMP_NETCONFIG contains `:force`. More importantly, the whole function is evaluated in a subshell, so the resulting assignment would not be propagated where it's needed. This means the whole check should be moved to another place and kdump_check_net should honour the previous value of kdump_neednet.

Reply:
What I originally meant: There check is repeated in `kdump_modify_config`, and if one is changed, the other one must be kept in sync. However, the check is now trivial enough to make that a non-issue.